src/main/kotlin/uk/gov/justice/digital/assessments/services/PredictorService.kt

```diff
@@ -0,0 +1,62 @@
+package uk.gov.justice.digital.assessments.services
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.assessments.api.AnswersDto
+import uk.gov.justice.digital.assessments.api.PredictorScoreDto
+import uk.gov.justice.digital.assessments.jpa.entities.AssessmentSchemaCode
+import uk.gov.justice.digital.assessments.jpa.entities.PredictorFieldMapping
+import uk.gov.justice.digital.assessments.jpa.entities.PredictorType
+import java.util.UUID
+
+@Service
+class PredictorService(
+  private val assessmentSchemaService: AssessmentSchemaService,
+) {
+  companion object {
+    val log: Logger = LoggerFactory.getLogger(this::class.java)
+  }
+
+  fun getPredictorResults(
+    assessmentSchemaCode: AssessmentSchemaCode,
+    episodeAnswers: Map<UUID, AnswersDto>,
+  ): List<PredictorScoreDto> {
+    val predictors = assessmentSchemaService.getPredictorsForAssessment(assessmentSchemaCode)
+
+    log.info("Found ${predictors.size} predictors for assessment type $assessmentSchemaCode")
```

Comment:
we could add the episode uuid here so we know which episode we are getting predictors for

Reply:
💯 - Added the Episode UUID to the log line